sharing-service/services/sharing/confirmInvite.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+const { getUser } = require('slic-tools/get')
+const { createResponse } = require('slic-tools/response')
+const share = require('./share')
+const { processEvent } = require('slic-tools/event-util')
+const { getSSMValue } = require('./lib/getSSMValue')
+const crypto = require('crypto')
+const { createShareAcceptedEvent } = require('slic-tools/event-dispatcher')
+
+async function main(event) {
+  const { body } = processEvent(event)
```

Comment:
Try and separate the Lambda handler from its implementation so we don't have tight coupling. See other services already implemented.